doc/tutorials/hybrid_gpu_qpu.ipynb

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
<p>You</p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>